server/service/src/invoice_line/inbound_shipment_service_line/update/mod.rs

```diff
@@ -5,18 +5,14 @@ use generate::generate;
 use repository::{InvoiceLine, InvoiceLineRowRepository, RepositoryError};
 use validate::validate;
 
-use crate::{
-    invoice_line::{query::get_invoice_line, ShipmentTaxUpdate},
-    service_provider::ServiceContext,
-    WithDBError,
-};
+use crate::{invoice_line::query::get_invoice_line, service_provider::ServiceContext, WithDBError};
 #[derive(Debug, Clone, PartialEq, Default)]
 pub struct UpdateInboundShipmentServiceLine {
     pub id: String,
     pub item_id: Option<String>,
     pub name: Option<String>,
     pub total_before_tax: Option<f64>,
-    pub tax: Option<ShipmentTaxUpdate>,
+    pub tax: Option<f64>,
```

Comment:
How to unset a tax value? With ShipmentTaxUpdate you where able to:
1) not touch the existing tax value in the update using `tax = None`
2) unset the tax value with `tax = Some(ShipmentTaxUpdate { percentage: None})`
3) update the tax value with `tax = Some(ShipmentTaxUpdate { percentage: Some(5.0)})`